build.gradle

```diff
@@ -3,6 +3,7 @@
 buildscript {
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
jcenter is a supserset of mavenCentral, it can do everything that mavenCentral can and more, so you don't need this